### ISSUE TITLE
title/bar button remains visible in ios11

### ISSUE THREAD

**Issue by eyvallah** [andreamazz/AMScrollingNavbar#257]
![simulator screen shot 2017 6 21 14 50 45](https://user-images.githubusercontent.com/17924287/27370970-5e16781a-5691-11e7-9f2f-fafce11754c4.png)

**Comment by andreamazz** [comment]
Awesome.  
I'll have to find the time to test this out, I'm a little swamped at the moment. 
As always, any help is deeply appreciated.

**Comment by eyvallah** [comment]
@andreamazz it seems to have sth to do with UIAppearance, by setting UINavigationBar.appearance().titleTextAttributes to [NSForegroundColorAttributeName: UIColor.white], the title issue is resolved. the navigationbar button issue is gone when i removed setting UIButton.appearance().tintColor in my code.

**Comment by andreamazz** [comment]
Hey @eyvallah 
I'm running the sample on iOS11 and everything seems fine. Can you confirm that it was resolved in the GM?

**Comment by yayoro** [comment]
I have same problem. :( How can i resolve it?

**Comment by andreamazz** [comment]
@yayoro did you try @eyvallah workaround? 
If it doesn't work, can you provide a sample showing the issue? Like I said the sample project seems to run fine.

**Comment by yayoro** [comment]
I try @eyvallah solution but not run correcly :(

![captura de pantalla 2017-09-21 a las 10 18 43](https://user-images.githubusercontent.com/26386809/30685814-747133e4-9eb6-11e7-8b86-0cc96db3af07.png)

**Comment by fullc0de** [comment]
@yayoro I had run into same problem. With several experiments, I ended up with finding a workaround without exact cause. Try set a tintColor to a navigationBar explicitly and remove a tintColor being set to individual `UIBarbuttonItem`. I hope my solution saves you :-)

**Comment by falsecrypt** [comment]
I had the same problem. You have to set the `tintColor` and `titleTextAttributes` in your `didFinishLaunchingWithOptions`:
```
        UINavigationBar.appearance().tintColor = .white
        UINavigationBar.appearance().titleTextAttributes = [NSAttributedStringKey.foregroundColor: UIColor.white]
```
Now everything works again.

**Comment by jwolkovitzs** [comment]
this is fixed in #278
